src/components/sidebar/Navbar.jsx

```diff
@@ -10,33 +10,33 @@ function Navbar() {
   return (
     <nav className="navbar">
       <ul>
-        <li>
+      <li>
         <NavLink 
-            to="/delivery" 
+            to="/navigation" 
             className={({ isActive}) => isActive ? "active" : "inactive"}
           >
-            <DeliveryIcon fontSize="large" className="navbar__icon" />
-            Delivery
+            <NavigationIcon fontSize="large" className="navbar__icon" />
+            Navigation
           </NavLink>
         </li>
 
         <li>
         <NavLink 
-            to="/servicing" 
+            to="/arm-dexterity" 
             className={({ isActive}) => isActive ? "active" : "inactive"}
           >
-            <ServicingIcon fontSize="large" className="navbar__icon" />
-            Servicing
+            <ArmDexterityIcon fontSize="large" className="navbar__icon" />
+            ArmDexterity
```

Comment:
Making this PascalCase looks a little weird on the UI, can we add a space and make the navlink text "Arm Dexterity" instead?